docs/usage/docker.md

```diff
@@ -265,6 +270,46 @@ Renovate can authenticate with AWS ECR using AWS access key id & secret as the u
 }
 ```
 
+##### Using the login password (basic auth)
+
+To authenticate using basic auth, simply provide the password in your config. For example:
+
+```json
+{
+  "hostRules": [
+    {
+      "hostType": "docker",
+      "matchHost": "12345612312.dkr.ecr.us-east-1.amazonaws.com",
+      "encrypted": {
+        "password": "w...A"
+      }
+    }
+  ]
+}
+```
+
+To obtain a login password, using the command `aws ecr get-login-password` or the corresponding API.
+
+##### Using an authorization token
+
+To authenticate using an authorization token, provide only the token in your config. For example:
+
+```json
+{
+  "hostRules": [
+    {
+      "hostType": "docker",
+      "matchHost": "12345612312.dkr.ecr.us-east-1.amazonaws.com",
+      "encrypted": {
+        "token": "QV...09"
+      }
+    }
+  ]
+}
+```
+
+To obtain an authorization token, using the command `aws ecr get-authorization-token` or the corresponding API.
```

Comment:
```suggestion
To get an authorization token, use the command `aws ecr get-authorization-token` or the corresponding API.
```

Same question here, do we expect AWS users to know about the API, or should we link to the docs?